src/main/java/reposense/parser/CsvParser.java

```diff
@@ -126,4 +137,22 @@ private static void setAliases(String[] elements, RepoConfiguration config, Auth
             config.setAuthorAliases(author, aliases);
         }
     }
+
+    private static RepoConfiguration buildRepoConfig(String location, String branch) throws ParseException {
+        if (PATH_PATTERN.matcher(location).matches()) {
```

Comment:
is there a need to use pattern for this? 
You can just use `Path.get` or `new URL(...)` if `Path` doesn't work isn't it?
Otherwise, we would have to test the Patterns as well.

Reply:
I'll go verify if `URL` is suffice for .git links.

Anyway, I believe these checks would be more appropriate if located in RepoConfiguration` constructor. Should I change it?